### ISSUE TITLE
UX Assurance - Flex Team Queue

### ISSUE THREAD

**Issue by arielleandrews** [bcgov/foi-flow#1094]
Columns in Flex Team Queue should match layout in design.

- [ ] First column on the left should be "ID Number"
- [ ] Next column "Applicant Name"
- [ ] Next column "Category"
- [ ] Next column "Current State"
- [ ] Next column "Analyst"
- [ ] "Type"
- [ ] "Days Left"
- [ ] "XGov"

Screenshot: 
![image.png](https://images.zenhubusercontent.com/608206453965c51946fdcabf/0a4aab92-2e74-40e6-848e-9611c2afc671)

Link to Design: 
https://jacklyn808742.invisionapp.com/console/share/3W2O64AP5C/831805474

**Comment by divyav-aot** [comment]
@arielleandrews  - If the user belongs to multiple groups then which dashboard to display by default. For example, a user belongs to Intake and Flex team. Which dashboard to show for this user?

**Comment by m-prodan** [comment]
Quick question on this - doesn't flex team want CFR Due date on their queue? Or did we hear otherwise? @JHarrietha-AOT @arielleandrews

**Comment by JHarrietha-AOT** [comment]
@m-prodan @arielleandrews Link above has been updated

**Comment by arielleandrews** [comment]
@divyav-aot users generally only belong to one group at a time but this is a good question. Any thoughts @m-prodan or @JHarrietha-AOT?

**Comment by JHarrietha-AOT** [comment]
I would agree, users should only belong to one user group so this shouldn't be an issue

**Comment by m-prodan** [comment]
Are we allowing for folks to be on multiple groups as users? I know we're trying to flatten the permissions of IAO users so they can do everything when they need to assist other teams, but I think their queue should reflect the group they are associated with.

**Comment by mpilchar** [comment]
Decision made at refinement Dec 17 for Super Users to have Processing Queue as their default queue - they'll be able to see all requests but their columns will be set up as processing queue users

**Comment by mpilchar** [comment]
@arielleandrews  Please rewrite as a user story and cross-check against #957 (duplicate).

**Comment by m-prodan** [comment]
Looks good - setting this as high priority as this is something we'll want to have in time for training

**Comment by richard-aot** [comment]
Increase estimate from 3 to 5. Need more time to implement default sorting (Open on top).

**Comment by arielleandrews** [comment]
@jadmsaadaot @richard-aot per discussion at bug triage this morning, all columns should be sortable. As it stands right now it does not appear the Days Left column is sortable. X-ref to bug ticket #1721

**Comment by m-prodan** [comment]
Main Functionality has passed as required. Of note, the bug ticket Arielle wrote up in #1721 will be handled separately. AC is passed, will close this ticket.
